server/controllers/loginUser.js

```diff
@@ -0,0 +1,39 @@
+const jwt = require('jsonwebtoken');
+const loginQuery = require('../database/queries/loginQuery');
+const { comparePassword } = require('../utils/hashPassword');
+const getHashPassword = require('../database/queries/getHashPassword');
+require('env2')('.env');
+
+const checkLogin = (req, res) => {
+  const { email, password } = req.body;
+  loginQuery(email)
+    .then((result) => {
+      if (result.rowCount) {
+        getHashPassword(email)
+          .then((hashed) => {
+            comparePassword(password, hashed.rows[0].pasword, (err, resu) => {
+              if (resu) {
+                jwt.sign({ is_Admin: true }, process.env.SECRET_KEY, (error, token) => {
```

Comment:
Don't mix promises with callback, 
make a file in your `utils` to convert `jwt.sign` to promise, So you can just call it in the next `.then()`